apps/siiges-app/pages/home/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Image from 'next/image';
+import { Container, Typography } from '@mui/material';
+import { ThemeProvider } from '@mui/material/styles';
+import { MainNavbar, PaperHome } from '@siiges-ui/shared';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Box from '@mui/material/Box';
+import theme from '../theme';
+
+export default function SignIn() {
+  return (
+    <ThemeProvider theme={theme}>
+      <Image
+        alt="travel"
+        src="/Fondo.jpg"
+        layout="fill"
+        objectFit="cover"
+        quality={100}
+        style={{
+          zIndex: -1,
+          overflow: 'hidden',
+        }}
+      />
```

Comment:
```suggestion
// file new-component.js
export default function NewComponent(src) {
  return (
   <Image
        alt="travel"
        src={src}
        layout="fill"
        objectFit="cover"
        quality={100}
        style={{
          zIndex: -1,
          overflow: 'hidden',
        }}
      />
  )
 }
 // home/index.js
 <Grid item xs={4} sx={{ marginTop: 7 }}>
   <NewComponent src='/avata.png'
```
**Note:** try to abstract ti more in order to make it more reusable